python/src/gen3authz/client/arborist/base.py

```diff
@@ -1,28 +1,22 @@
-"""
-Define the ArboristClient class for interfacing with the arborist service for
```

Comment:
looks like this might be wrong/old, but I do like module level docstrings when applicable (e.g. what's in this module) up to you if you want to add a relevant one

Reply:
This line is not changing - it is git who thinks the `base.py` is the new `client.py`. But I'll add something to `base.py`.